src/configuration/customers/requisition-lists.md

```diff
@@ -3,6 +3,18 @@ b2b_only: true
 title: Requisition Lists
 ---
 
+The Magento_RequisitionList module allows a customer to create multiple lists of frequently-purchased items and use those lists for order placement.
```

Comment:
@dnahirna , could you please replace `The Magento_RequisitionList module` with `The feature`.

Reply:
done